### PR TITLE
Copy fixes for docs

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

File: docs/server/upgrading.rst [hunk 1]
```diff
@@ -210,8 +210,9 @@ statistical data. You will need to have an :ref:`RQ worker running
 
 This command will dispatch jobs to the RQ worker and may take some time.
 
-If you wish to run ``calculate_checks`` and ``refresh_stats`` in the foreground
-without using the RQ worker you can use the :option:`--no-rq` option.
+If you wish to run :djadmin:`calculate_checks` and :djadmin`refresh_stats` in
```
Comment (julen) [inline docs/server/upgrading.rst]: You have a missing semicolon in the second `:djadmin:`.

Comment (phlax) [inline docs/server/upgrading.rst]: thanks

File: docs/server/requirements.rst [hunk 2]
```diff
@@ -74,6 +74,9 @@ To install and run Redis on a Debian-based system:
    $ sudo apt-get install redis-server
    $ sudo services redis-server start
 
+Pootle requires a minimum Redis server version of 2.8.4. If you are using
+Debian Wheezy you will need to install `redis-server` from backports.
```
Comment (unho) [inline docs/server/requirements.rst]: Shouldn't this be a note to grab the readers eye?

Comment (phlax) [inline docs/server/requirements.rst]: sure - ill add

File: docs/server/installation.rst [hunk 3]
```diff
@@ -107,7 +107,7 @@ To verify that everything installed correctly, you should be able to access the
 .. code-block:: bash
 
   (env) $ pootle --version
-  Pootle 2.7.1 (Django 1.7.10, Translate Toolkit 1.13.0)
+  Pootle 2.7.2 (Django 1.7.10, Translate Toolkit 1.13.0)
```
Comment (unho) [inline docs/server/installation.rst]: This change rather belongs to docs adjustment when 2.7.2 release is being done, rather to a general docs cleanup, isn't it?

I prefer this specific change to be moved outside of this PR, so these docs cleanups can be cherry picked to the stable/2.7.1 branch.

Comment (phlax) [inline docs/server/installation.rst]: fine

**Comment by unho** [comment]
lgtm. Please backports these changes to stable/2.7.1 as well.

**Pull request merged**